### PR TITLE
Add the Statusify module and the Config submodule.

### PR DESCRIPTION
**Pull request by ur0**
This makes it easier to load application configuration, uses a global `Statusify` module instead of using application-wide constants.

### CODE REVIEW

File: lib/statusify/config.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    def app_name
+      # The application's name
+      # Attempts to read from both - the config file and the environment
+      (YAML_CONFIG['name'] || ENV['APP_NAME']).capitalize
+    end
+
+    def theme
+      # The current theme we're using
+      YAML_CONFIG['theme'] || 'flat-ui'
```
Comment (justanshulsharma) [inline lib/statusify/config.rb]: you shall use 'flat-ui'.freeze

File: lib/statusify/statusify.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module Statusify
+  # Our module. Does awesome stuff like loading config.
+  require_relative '../../lib/statusify/config'
+  include Config
```
Comment (justanshulsharma) [inline lib/statusify/statusify.rb]: why are you using include and extend both ?

Comment (ur0) [inline lib/statusify/statusify.rb]: Weird typo. RubyMine messed up. Thanks for pointing out.

Comment (justanshulsharma) [inline lib/statusify/statusify.rb]: haha np ;)

File: lib/statusify/config.rb (hunk 1 above)
Comment (ur0) [inline lib/statusify/config.rb]: I'm not freezing now since I plan to add a feature to switch themes in the demo.

**Pull request merged**